src/script_chainer/gui/page/script_edit_interface.py

```
@@ -274,14 +310,39 @@ def on_script_path_chosen(self, file_path) -> None:
+    def _sync_launch_method_ui(self, sync_expand: bool = False) -> None:
+        launcher_mode = self.launcher_mode_switch.isChecked()
+        self.script_process_name_opt.setEnabled(launcher_mode)
+        if sync_expand and launcher_mode:
+            self.launch_advanced_group.setExpand(True)
+
+    def _sync_launch_advanced_summary(self, *_args) -> None:
+        config = self.get_config_value()
+        if config.launcher_mode:
+            target_name = config.script_process_display_name
+            if target_name:
+                summary = f'当前会等待启动后的主程序：{target_name}'
```

Ruff (0.15.13)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)